### ISSUE TITLE
Location module

### ISSUE THREAD

**Issue by importshark** [leon-ai/leon#439]
The intent of this module is to be able to ask Leon to suggest nearby theaters, restaurants, and he can.

### Feature Use Case

Leon, recommend a nearby café with a rating above four

I get that Leon is intended to be safe, offline, and secure, so it would make sense if this would not be a feature to be added at this time. Thanks a lot for considering my feature, ~Shark

**Comment by AthithyaIthayan** [comment]
@importshark  adding near by hotels can be through google maps API then that's a whole lot of improvement 
Is that what u mean or any other way to work with?

**Comment by importshark** [comment]
Yeah essentially that. Being able to get the current location and access information about nearby businesses, parks, etc.

**Comment by AthithyaIthayan** [comment]
@importshark Its great but the API key Needs to be generated right ? Its a paid process any Idea to work over?

**Comment by importshark** [comment]
@AthithyaIthayan yeah that's the problem. We can probably talk more about this when the core is rewritten

**Comment by AthithyaIthayan** [comment]
code?
Ig we can at least let him open google maps through a command as of now?
